api-mock/fetcher.ts

```diff
@@ -0,0 +1,248 @@
+import { fetch } from "cross-fetch";
+import { MockedVars } from "./api-mock-server";
+import { PROCESS_STATE } from "supertokens-node/lib/build/processState";
+import { serializeVars } from "./utils";
+
+const apiMockPort = process.env.ST_SDK === "golang" ? 3032 : process.env.ST_SDK === "python" ? 3031 : 3030;
+
+let status: "NOT_READY" | "OK" = "NOT_READY";
```

Comment:
```suggestion
let apiStatus: "NOT_READY" | "OK" = "NOT_READY";
```

Reply:
it's easy to accidentally run into shadowing issues with it being just `status`

Reply:
agreed, changed